### PR TITLE
add cli kwargs for image_quality and frame_step when creating task

### PR DESCRIPTION
**Pull request by qztseng**
### Motivation and context

When using CLI to upload images to create tasks, we would like to have control over some advanced configuration like frame step and image quality. 

### How has this been tested?

It has been tested using the tasks_create API by including additional kwargs (image_quality, frame_step):
`CLI.tasks_create(cli, name=name, labels=[], overlap=0, segment_size=0,
                             resource_type=ResourceType.argparse('local'),
                             resources=images, image_quality=iq, frame_step=frame_step,
                             project_id=pid, bug='')`

Maybe someone can help creating a test script ?

### Checklist

- [ X] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
@qztseng Thanks for the contribution!
Could you please add the corresponding options into argument parser?
Also make sure these changes work, I don't see you changing the call of the `tasks_data` function

**Comment by qztseng** [comment]
> @qztseng Thanks for the contribution!
> Could you please add the corresponding options into argument parser?
> Also make sure these changes work, I don't see you changing the call of the `tasks_data` function
I just fixed it and tested. Additional options also added to the parser.

**Comment by bsekachev** [comment]
@qztseng 

Please, add the information into CHANGELOG.md and update PR header (check necessary checkboxes, like license if you agree submit your code under MIT license).

@azhavoro 

Could you please look at the PR?

File: utils/cli/core/core.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2020 Intel Corporation
+# Copyright (C) 2021 Intel Corporation
```
Comment (bsekachev) [inline utils/cli/core/core.py:1]: It must be 2020-2021

Comment (qztseng) [reply utils/cli/core/core.py]: changed to 2020-2021

**Comment by azhavoro** [comment]
@qztseng Hi, I found an issue:
```python
 python cli.py --auth user:pass --server-host xxx create --help
Traceback (most recent call last):
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 42, in <module>
    main()
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 28, in main
    args = parser.parse_args()
  File "/usr/lib/python3.9/argparse.py", line 1818, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/lib/python3.9/argparse.py", line 1851, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.9/argparse.py", line 2042, in _parse_known_args
    positionals_end_index = consume_positionals(start_index)
  File "/usr/lib/python3.9/argparse.py", line 2019, in consume_positionals
    take_action(action, args)
  File "/usr/lib/python3.9/argparse.py", line 1928, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib/python3.9/argparse.py", line 1207, in __call__
    subnamespace, arg_strings = parser.parse_known_args(arg_strings, None)
  File "/usr/lib/python3.9/argparse.py", line 1851, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.9/argparse.py", line 2060, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib/python3.9/argparse.py", line 2000, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib/python3.9/argparse.py", line 1928, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib/python3.9/argparse.py", line 1092, in __call__
    parser.print_help()
  File "/usr/lib/python3.9/argparse.py", line 2548, in print_help
    self._print_message(self.format_help(), file)
  File "/usr/lib/python3.9/argparse.py", line 2532, in format_help
    return formatter.format_help()
  File "/usr/lib/python3.9/argparse.py", line 283, in format_help
    help = self._root_section.format_help()
  File "/usr/lib/python3.9/argparse.py", line 214, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.9/argparse.py", line 214, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.9/argparse.py", line 214, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.9/argparse.py", line 214, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.9/argparse.py", line 530, in _format_action
    help_text = self._expand_help(action)
  File "/usr/lib/python3.9/argparse.py", line 626, in _expand_help
    return self._get_help_string(action) % params
ValueError: unsupported format character ')' (0x29) at index 111
```

**Comment by qztseng** [comment]
> @qztseng Hi, I found an issue:
> 
> ```python
>  python cli.py --auth user:pass --server-host xxx create --help
>...
problem should be solved......

**Comment by azhavoro** [comment]
LGTM, thanks for the contribution!

**Review by azhavoro — APPROVED** [review]

**Pull request merged**